### ISSUE TITLE
Improve the design

### ISSUE THREAD

**Issue by Shanky1001** [Shanky1001/Imagine-AI#1]
Try to improve the UI of the pages

**Comment by Sarvaswa-Mohata** [comment]
Hi, I would like to get this issue assigned to me

**Comment by Shanky1001** [comment]
> Hi, I would like to get this issue assigned to me

Sure @Sarvaswa-Mohata , Good luck

**Comment by Sarvaswa-Mohata** [comment]
Hi, I made changes in a figma file for the landing page. I am attaching a screenshot of the same below :
![Group 4](https://github.com/Shanky1001/Imagine-AI/assets/99800509/b7425308-5a71-44c3-a4ed-4e853647f903)

If you approve of the changes, I'll proceed with coding the same.

**Comment by Shanky1001** [comment]
@Sarvaswa-Mohata 

Form is looking good. Please provide the figma link as the place for the generated image is not getting clear to me from this image.

I have only a few suggestions -

- I guess you missed the prompt generate button on the second input field.
- background image is taking too much of focus rather than form so managing that will be great.

**Comment by Sarvaswa-Mohata** [comment]
Hi, @Shanky1001 
I was planning to do it this way:
![image](https://github.com/Shanky1001/Imagine-AI/assets/99800509/af3d227d-10d0-4443-ac43-0a05da2ec5da)

where if a person hovers over the image, he/she can download it. So, the car image is not a background image, rather it is assumed to be a generated image based on the given prompt. 

I am also attaching the figma file below. Apart from this I plan on showing a set of similar images on the sidebar, but that totally depends on whether your ai generator can handle it. A design implementation of the same can be seen if you open up the figma file. Looking forward to your comments :) 

[Link to the figma file](https://www.figma.com/file/Kpu9OoZzgQMRL4PLunicf3/Imagine-AI?type=design&node-id=0%3A1&mode=design&t=kI0uXb0Ja5pCCTLE-1)

**Comment by Shanky1001** [comment]
hi @Sarvaswa-Mohata ,

looks good to me. Go ahead, give it a try.

**Comment by parinitapatil** [comment]
@Shanky1001, I recommend implementing _UI improvements_, particularly with regards to **padding** and **borders**. Let me know if you need any help. Kind regards.

**Comment by Shanky1001** [comment]
@parinitapatil What kind of UI improvements regarding padding and border, you are suggesting? 
if you can provide a bit more info on that it will be helpful in understanding what exactly you want to do.

**Comment by parinitapatil** [comment]
> @parinitapatil What kind of UI improvements regarding padding and border, you are suggesting? if you can provide a bit more info on that it will be helpful in understanding what exactly you want to do.

I have identified areas that require improvement:
1. The buttons need to be resized. 
2. The colors are too dull.
3. The input labels are not aligned with the UI.

**Comment by Shanky1001** [comment]
@parinitapatil Okay sounds good, Give it a try.

but in my opinion, it will be better if you and @Sarvaswa-Mohata can collaborate over his figma and implement the design together.

**Comment by parinitapatil** [comment]
> @parinitapatil Okay sounds good, Give it a try.
> 
> but in my opinion, it will be better if you and @Sarvaswa-Mohata can collaborate over his figma and implement the design together.

![Screenshot 2023-10-02 121727](https://github.com/Shanky1001/Imagine-AI/assets/119065651/bd910d85-22b4-43f1-ae25-73b82552784c)
I hope this is helpful to you.

**Comment by Shanky1001** [comment]
> > @parinitapatil Okay sounds good, Give it a try.
> > 
> > but in my opinion, it will be better if you and @Sarvaswa-Mohata can collaborate over his figma and implement the design together.
> 
> ![Screenshot 2023-10-02 121727](https://github.com/Shanky1001/Imagine-AI/assets/119065651/bd910d85-22b4-43f1-ae25-73b82552784c)
> I hope this is helpful to you.

Yeah, this looks a bit more intuitive. let's get it done.
 @Sarvaswa-Mohata @parinitapatil .
We can later have other design changes if required.

**Comment by parinitapatil** [comment]
Sir, do you happen to have any remaining Hacktoberfest projects? It's proving to be quite challenging to locate them.

**Comment by Shanky1001** [comment]
> Sir, do you happen to have any remaining Hacktoberfest projects? It's proving to be quite challenging to locate them.

Yeah, you can try this project [Apex Legend Clone](https://github.com/Shanky1001/Apex_Legends_Clone/issues)
you can also raise issues on your own if find any.

Or you can search hacktoberfest keyword on github search and filter it using tags, you will get the repos. Just make sure they are in hacktoberfest2023.

**Comment by Sarvaswa-Mohata** [comment]
Hi @parinitapatil, do you wanna collaborate over this project?

**Comment by parinitapatil** [comment]
yeh sure!!

**Comment by Sarvaswa-Mohata** [comment]
@parinitapatil can we connect via discord?, I think it would be better to communicate over there.

**Comment by GauraviVeling** [comment]
Hello. Please can you assign this issue to me. I would like contributing to this issue.

**Comment by Shanky1001** [comment]
> Hello. Please can you assign this issue to me. I would like contributing to this issue.

sure, go ahead and fix the issue. It will be assigned.
